src/DatabaseLibrary/query.py

```diff
@@ -521,13 +671,20 @@ def call_stored_procedure(
                         result_sets.append(list(result_set))
 
             else:
-                logger.info(
-                    f"CAUTION! Calling a stored procedure for '{db_connection.module_name}' is not tested, "
-                    "results might be invalid!"
-                )
-                cur = db_connection.client.cursor()
+                if db_connection.module_name == "pymssql":
+                    mssql = importlib.import_module("pymssql")
+                    spParams = spParams.copy()
+                    for param in additional_output_params:
```

Comment:
Note that output params always being spread into the param list at the end of the list means that only SQL stored procedures which have their outputs at the end are usable here. That's generally best practice for writing a stored procedure, but do you want to be requiring library users to refactor their stored procedures to follow best practices in order to be testable?

Reply:
Hmm, this is a good point.
Do you mean a case when a procedure takes, let's say, 4 params and they're not listed like `IN, IN, OUT, OUT`, but `ÌN, OUT, IN, OUT`?
Do you have any ideas how to solve it?